### PR TITLE
AMP4Email ban amp-form support for submitting state

### PR DESCRIPTION
**Pull request by alabiaga**
Rather than supporting this intermediary state of a form submit and dealing with the additional RPC call, we are banning it for AMP4Email.

### CODE REVIEW

**Comment by alabiaga** [comment]
cc\ @GoTcWang

File: validator/validator-main.protoascii [hunk 1]
```diff
@@ -26,7 +26,7 @@ min_validator_revision_required: 328
 # newer versions of the spec file. This is currently a Google internal
 # mechanism, validator.js does not use this facility. However, any
 # change to this file (validator-main.js) requires updating this revision id.
-spec_file_revision: 662
+spec_file_revision: 658
```
Comment (honeybadgerdontcare) [inline validator/validator-main.protoascii]: spec file revision shouldn't change

File: validator/validator-main.protoascii [hunk 2]
```diff
@@ -3509,7 +3511,7 @@ tags: {
   attr_lists: "input-name-attr"
   # Note that this amp-bind attribute is not available for the AMP4Email format
   # thus we add this manually for the formats that should.
-  attrs: { name: "[type]" }
+  attrs: {name: "[type]"}
```
Comment (honeybadgerdontcare) [inline validator/validator-main.protoascii]: please undo this change as well

Comment (alabiaga) [reply validator/validator-main.protoascii]: Same here. not sure what happened here. reverted. perhaps my editor formatting style. reverted.

File: validator/validator-main.protoascii (hunk 1 above)
Comment (alabiaga) [reply validator/validator-main.protoascii]: i didn't change intentionally. Not sure how this happened. reverted.

**Comment by alabiaga** [comment]
Thanks for the prompt review David!

File: validator/validator-main.protoascii [hunk 3]
```diff
@@ -3252,6 +3252,8 @@ tags {  # <form method=POST ...>
   attrs: { name: "novalidate" }
 }
 tags {
+  html_format: AMP
+  html_format: AMP4ADS
```
Comment (choumx) [inline validator/validator-main.protoascii]: Add `html_format: EXPERIMENTAL`.

Comment (alabiaga) [reply validator/validator-main.protoascii]: done. thanks!
